app/models/lesson.rb

```diff
@@ -8,8 +8,12 @@ class Lesson < ApplicationRecord
   belongs_to :author, class_name: 'User'
   enum :status, STATUSES
 
-  validates :title, presence: true
-  validates :description, presence: true
-  validates :video_link, presence: true
+  validates :title, presence: true, length: { maximum: 64, too_long: 'The maximum allowed number of character is 64' },
+                    format: { with: %r/\A[а-яА-ЯёЁa-zA-Z0-9`:%&'"@,><{|}=_~\-!()$*+\/?\^.\[\]]+\z/ }
```

Comment:
Notes:
1. Please add space character (**\s**) both to the 'title' and 'description' formats
2. Please add '**:**' character for the description (I think it is a mistake in the requirements). And leave a comment for QA about it in the ticket
3. Do we need message for the format in '**title**'?
4. In case of reqular expressions are equal both for the 'title' and 'description' you can create a separate constant for it

Reply:
In requirements It was nothing said about the error message for title

Reply:
Ok